### ISSUE TITLE
FLARM limitations hardcoded?

### ISSUE THREAD

**Issue by JEK58** [fhorinek/BB#3]
Are the FLARM limitations (e.g. it only shows hang/paragliders) hardcoded to the FLARM module or will it be possible to change this behaviour with custom code?

**Comment by fhorinek** [comment]
You can only broadcast your position via FLARM. Yo can see other hang/paraqgliders via FANET network.
The limitation is based on licence agreement with FLARM, the FANET+ module runs proprietary encrypted firmware, to be able to broadcast to FLARM network. You can choose to run fully open-source but without the FLARM beacon capability.

**Comment by Tokter** [comment]
The image here https://www.facebook.com/skybeanvario/photos/pcb.1921213768044520/1921212951377935/ shows the Skytraxx module connected to a Pulse W3329 antenna. Doesn't that limit it to only 868MHz? So that no software update will ever make FLARM or FANET useable outside of Europe?

**Comment by fhorinek** [comment]
Currently the FANET module software does not support 915Mhz band, however the module hardware does support he band as well as our antenna on strato. Our antenna can be used, according to our measurement for 915MHz as well as 868MHz.
915MHz band will be enabled as soon as the module supports it.

From the FANET+ module datasheet:
![image](https://user-images.githubusercontent.com/9072684/116053184-adfa1200-a67a-11eb-99bb-9ea69ff788ed.png)

**Comment by kolupaev** [comment]
Waiting on 915Mhz Flarm support as well. Would be awesome to improve PG visibility. 
Could you share link to the FANET+ module spec or changelog so we can ask for or track the issue?

**Comment by fhorinek** [comment]
I did send the request to the module developer (SkyTraxx).
They told me that the 915MHz (US) and 920HMz (Jap/Aus) is planned, but because of the disruption of the supplier chain and problems caused by it it is currently postponed. 
(My guess is that they need to adapt their product to different platform.)

Here is the module repo https://github.com/3s1d/fanet-stm32

**Comment by kolupaev** [comment]
Looks like fanet is targeting 920 for North Americas, which is great!

Would Strato support that? 

Commit id: 
https://github.com/3s1d/fanet-stm32/commit/13b5568c0bb3163d45878fbbc8a255e67f64f86d

**Comment by fhorinek** [comment]
yes it will, we are waiting for approved fanet firmware
